### PR TITLE
[AST] NFC: Remove {TokenKinds, Attr}.def.gyb from CMakeLists

### PR DESCRIPTION
**Pull request by xedin**

### CODE REVIEW

**Comment by xedin** [comment]
@swift-ci please clean test

File: include/swift/AST/CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,5 @@ else()
 endif()
 
 set(generated_include_sources
-    Attr.def.gyb
     TokenKinds.def.gyb)
```
Comment (ahoppen) [inline include/swift/AST/CMakeLists.txt:8]: `TokenKinds.def.gyb` doesn’t exist anymore either. I removed both of them in https://github.com/apple/swift/pull/67666

Comment (xedin) [reply include/swift/AST/CMakeLists.txt:8]: Perfect, I'll remove that as well!

Comment (xedin) [reply include/swift/AST/CMakeLists.txt]: ✅

**Comment by xedin** [comment]
@swift-ci please clean test

**Review by ahoppen — APPROVED** [review]
Thanks

**Pull request merged**